pkg/api/middlewares.go

```diff
@@ -34,7 +34,7 @@ func (s *Server) newAPILogger(next http.Handler) http.Handler {
 		t1 := time.Now()
 		defer func() {
 			logger.APIAccess(r.Method, r.RemoteAddr, r.URL.Path, ww.Status(),
-				r.Response.ContentLength, int64(ww.BytesWritten()),
+				0, int64(ww.BytesWritten()),
```

Comment:
In `logger.APIAccess` function, the fifth parameter is `bodyBytedReceived`.  So the original implementation, this parameter is valued with `ctx.ContentLgnth` which returns the request's header value of "Content-Length".[1]
So I think using `r.ContentLength` here is definitely better than just valued it with `0`. 

## Reference
[1] https://github.com/megaease/easegress/blob/main/pkg/api/middlewares.go#L53